### ISSUE TITLE
DataView filter the left and right axis drop downs to only include columns with data

### ISSUE THREAD

**Issue by anchapin** [SEED-platform/seed#3520]
I was getting errors when selecting a column that didn't include data and had to go to inventory page and double check which columns had data and then go back to Custom Reports and update the column selected. It would be nice if only columns with data were available to select.

**Comment by isalanglois** [comment]
Waiting on user feedback

**Comment by RDmitchell** [comment]
@isalanglois -- I added myself to this issue and set it to Test, which I will do next week.

**Comment by anchapin** [comment]
@RDmitchell, I don't believe that this has been implemented yet. Therefore, I'm not sure why it was closed but I don't think there's anything for you to test yet. I could be wrong but I don't remember this being worked on.

**Comment by RDmitchell** [comment]
@anchapin -- I moved it to the Q2 project under To Do

**Comment by isalanglois** [comment]
Talked with @anchapin and this is a long term item, not for Q2. Moving to Q4 board
